### ISSUE TITLE
损失函数

### ISSUE THREAD

**Issue by yangkang779** [bilylee/SiamFC-TensorFlow#13]
loss = tf.nn.sigmoid_cross_entropy_with_logits(logits=response,
                                                     labels=gt)
这个函数在全卷积中的形式是什么样呀，一维的看的明白，这是在【8，15，15】还有【8，17，17】的维度上直接用的，如果把人脸中的center loss加进来的话，这样的函数维度应该怎么匹配啊？最近没看明白，centerl  loss函数该怎么加啊。谢谢作者，没人讨论，就自己在看

**Comment by bilylee** [comment]
tf.nn.sigmoid_cross_entropy_with_logits 是对 logits elementwise 求 loss，你可以理解成是先将 [8, 15, 15] resize 到一维，然后再算 loss。

如果我的理解没错的话，center loss 需要对每个视频中的每个 object 学一个 center，然后算训练样本的feature 与其 center 的 L2 距离。
